app/views/publishers/jobseeker_profiles/_layout.html.slim

```diff
@@ -21,4 +21,5 @@
             data-clipboard-target="button"
             class="js-action govuk-button govuk-button--secondary govuk-!-margin-bottom-2 clipboard-copy"]
         | Copy email
+  .govuk-grid-column-two-thirds
```

Comment:
Non-blocking question: Shouldn't this 2/3 column be nested under a separate/new row?

In the original layout, we had everything under a 2/3 column in a single row. Now we have 2/3 column and a full-size column together within the same row.
Combining 2/3 + Full in a single row seems contradictory.  So given that this is going to be displayed below of the full-size column content, it seems to belong to a different row?